### PR TITLE
docs(components): [image] add type of crossorigin

### PR DESCRIPTION
**Pull request by besscroft**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/image.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ image/image-preview
...
 | alt                   | native attribute `alt`.                                                                                                                           | ^[string]                                                               | —       |
 | referrerpolicy        | native attribute [referrerPolicy](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/referrerPolicy).                              | ^[string]                                                               | —       |
-| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` | 'anonymous'                                                             | —       |
+| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials' \| 'null'`               | 'anonymous'                                                             | —       |
```
Comment (kooriookami) [inline docs/en-US/component/image.md:72]: Don't change this, please clear the default value of `crossorigin`.

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: docs/en-US/component/image.md (hunk 1 above)
Comment (kooriookami) [reply docs/en-US/component/image.md:72]: https://github.com/element-plus/element-plus/blob/dev/packages/components/image/src/image.ts#L111C26-L111C26

Please remove this value.

File: docs/en-US/component/image.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ image/image-preview
...
 | alt                   | native attribute `alt`.                                                                                                                           | ^[string]                                                               | —       |
 | referrerpolicy        | native attribute [referrerPolicy](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/referrerPolicy).                              | ^[string]                                                               | —       |
-| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` | 'anonymous'                                                             | —       |
+| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` | ''                                                             | —       |
```
Comment (kooriookami) [inline docs/en-US/component/image.md:72]:
```suggestion
| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` | —                                                             | —       |
```

File: packages/components/image/src/image.ts [hunk 3]
```diff
@@ -108,7 +108,7 @@ export const imageProps = buildProps({
    */
   crossorigin: {
     type: definePropType<'anonymous' | 'use-credentials' | ''>(String),
-    default: 'anonymous',
+    default: '',
```
Comment (kooriookami) [inline packages/components/image/src/image.ts:111]:
```suggestion
```
Refer to https://developer.mozilla.org/zh-CN/docs/Web/HTML/Attributes/crossorigin
Just delete the line is OK.

`将属性名称设置为空值，如 crossorigin 或 crossorigin=""，与设置为 anonymous 的效果一样。
不合法的关键字或空字符串会视为 anonymous 关键字。`

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: docs/en-US/component/image.md (hunk 2 above)
Comment (kooriookami) [inline docs/en-US/component/image.md:72]:
```suggestion
| crossorigin        | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                              | ^[enum]`'' \| 'anonymous' \| 'use-credentials'` \| ''                                                             | —       |
```
Is it missing a `\`?

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: docs/en-US/component/image.md (hunk 2 above)
Comment (kooriookami) [inline docs/en-US/component/image.md:72]: | ''   | —       |
There's an extra space.

File: docs/en-US/component/image.md [hunk 4]
```diff
@@ -59,26 +59,26 @@ image/image-preview
...
+| scroll-container      | the container to add scroll listener when using lazy load. By default, the container to add scroll listener when using lazy load.                 | ^[string] / ^[object]`HTMLElement`                                      | —         |
+| alt                   | native attribute `alt`.                                                                                                                           | ^[string]                                                               | —         |
+| referrerpolicy        | native attribute [referrerPolicy](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/referrerPolicy).                              | ^[string]                                                               | —         |
+| crossorigin           | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                                         | ^[enum]`'' \| 'anonymous' \| 'use-credentials'`                         | ''        |
```
Comment (kooriookami) [inline docs/en-US/component/image.md:72]:
```suggestion
| crossorigin           | native attribute [crossorigin](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).                                         | ^[enum]`'' \| 'anonymous' \| 'use-credentials'`                         | —         |
```
One small thing.

**Review by kooriookami — CHANGES_REQUESTED** [review]

**Comment by besscroft** [comment]
Sorry, it was my mistake. I asked IDEA to format it in the same table style below and that was it.

**Comment by kooriookami** [comment]
Oops, the test failed. I think it's the problem of type.

**Review by kooriookami — APPROVED** [review]
LGTM

**Pull request merged**